packages/tsc/index.ts

```diff
@@ -46,3 +47,14 @@ export function run(tscPath = require.resolve('typescript/lib/tsc')) {
 		}
 	}
 }
+
+function getTscPath() {
+	const version = require('typescript/package.json').version as string;
+
+	if (semver.gte(version, '5.7.2')) {
```

Comment:
Shouldn't this include whole 5.7 instead?

```suggestion
	if (semver.gte(version, '5.7')) {
```